notebooks/book2/15/logreg_iris_bayes_2d.ipynb

```diff
@@ -1,49 +1,229 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    jax.vmap(joint_logprob_fn_partial)(initial_params)
</pre><p>This all seems much more complex than the pymc code. I wonder if BIJAX could be used to simplify this boilerplate?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Ohh, actually we don't need this line, this was just for sanity check. I've removed it.</p><p> </p><p>I think this is too simple example for BIJAX,  would it make more sense to show usage of BIJAX in little more complex example like hierarchical one where we needed bijectors?  I'll add BIJAX implementation in those examples.</p><p> </p>